articles/114_generated_interfaces_python.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title: Generated Python interfaces
+permalink: articles/generated_interfaces_python.html
+abstract:
+  This article describes the generated Python code for ROS 2 interfaces.
+published: true
+author: '[Dirk Thomas](https://github.com/dirk-thomas)'
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Scope
+
+This article specifies the generated Python code for ROS interface types defined in the [interface definition article](interface_definition.html).
+
+## Namespacing
+
+All code of a ROS package should be defined in a namespace named after the package.
```

Comment:
Should we use the term `namespace` here or the Pythonic term `package`? Perhaps we should continue to use `namespace`, but clarify that what we mean is within in a Python package there is a `msg` and/or `srv` sub-package in which there is a Python module for each message or service which contains a Python class.